bin/pycbc_make_sky_grid

```diff
@@ -156,12 +135,19 @@ parser.add_argument(
     help="List of instruments to analyze.",
 )
 parser.add_argument(
-    '--sky-error',
-    type=angle_as_radians,
+    "--input-dist",
+    type=str,
+    default='UniformSky()',
     required=True,
-    help="3-sigma confidence radius of the external trigger error "
-    "box. Use the rad or deg suffix to specify units, otherwise "
-    "radians are assumed.",
+    help="Input distribution of the sky map that you have, could "
+    "be HealpixSky or FisherSky. See sky location documentation for "
+    "more details",
+)
+parser.add_argument(
+    '--coverage',
+    type=float,
+    required=True,
+    help="Percentage of probability that you want to cover",
```

Comment:
Coverage will not make sense if the input distribution is `UniformSky`. I suggest you do the following:
1. Make `--input-dist` and `--coverage` optional arguments (remove `required=True`).
2. When instantiating `input_dist` and `sky_dist`, check if `--input-dist` and `--coverage` were given. If so, do what is being done now. Otherwise, instantiate both `input_dist` and `sky_dist` from `UniformSky`.